theme/css/chrome.css

```diff
@@ -0,0 +1,566 @@
+    --color-on-secondary-container: #fcddbd;
+    --color-tertiary: #bbcd9e;
+    --color-on-tertiary: #273513;
+    --color-tertiary-container: #3d4c28;
+    --color-on-tertiary-container: #d7e9b9;
+    --color-error: #ffb4ab;
+    --color-on-error: #690005;
+    --color-error-container: #93000a;
+    --color-on-error-container: #ffdad6;
+    --color-background: #18120c;
+    --color-on-background: #eee0d4;
+    --color-surface: #18120c;
+    --color-on-surface: #eee0d4;
+    --color-surface-variant: #504539;
+    --color-on-surface-variant: #d4c4b5;
+    --color-outline: #9c8e80;
+    --color-outline-variant: #504539;
+    --color-shadow: #000000;
+    --color-scrim: #000000;
+    --color-inverse-surface: #eee0d4;
+    --color-inverse-on-surface: #372f27;
+    --color-inverse-primary: #825513;
+    --color-primary-fixed: #ffddb8;
+    --color-on-primary-fixed: #2a1700;
+    --color-primary-fixed-dim: #f8bb71;
+    --color-on-primary-fixed-variant: #653e00;
+    --color-secondary-fixed: #fcddbd;
+    --color-on-secondary-fixed: #281805;
+    --color-secondary-fixed-dim: #dfc2a2;
+    --color-on-secondary-fixed-variant: #57432b;
+    --color-tertiary-fixed: #d7e9b9;
+    --color-on-tertiary-fixed: #121f02;
+    --color-tertiary-fixed-dim: #bbcd9e;
+    --color-on-tertiary-fixed-variant: #3d4c28;
+    --color-surface-dim: #18120c;
+    --color-surface-bright: #403830;
+    --color-surface-container-lowest: #130d07;
+    --color-surface-container-low: #211a13;
+    --color-surface-container: #251e17;
+    --color-surface-container-high: #302921;
+    --color-surface-container-highest: #3b332b;
+
+    /* Stuff */
+    --menu-bar-height: 6rem;
+    --sidebar-width: 36rem;
+    --font-family: 'Rubik', sans-serif;
+    --content-max-width: 80rem;
+}
+
+
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--color-surface-container-low);
+    font-family: var(--font-family);
+}
+
+.flex-spacer {
+    flex-grow: 1;
+}
+
+.page-wrapper {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-surface-container-low);
+}
+
+.page-inner {
+    min-height: 100%;
+    display: flex;
+    flex-direction: row;
+    overflow: visible;
+}
+
+.page {
+    flex: 1;
+    margin-top: 0;
+    padding: 2rem;
+    overflow: hidden;
+    background-color: var(--color-background);
+    border-radius: 0 0 0 0;
+}
+
+@media (max-width: 1000px) {
+    .page {
+        width: 100vw;
+        min-width: 100vw;
+        filter: brightness(1);
+    }
+
+    #sidebar-toggle-anchor:checked + .page-wrapper .page {
+        filter: brightness(0.8);
+    }
+}
+
+.no-js .page {
+    transition: border-radius 0.3s cubic-bezier(0.16, 1, 0.3, 1), filter 0.3s cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.sidebar-loaded .page {
+    transition: border-radius 0.3s cubic-bezier(0.16, 1, 0.3, 1), filter 0.3s cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+/* When checked */
+#sidebar-toggle-anchor:checked + .page-wrapper .page {
+    border-radius: 2rem 0 0 0;
```

Comment:
This radius doesn't persist when the page is scrolled, which is a little weird IMO.

Reply:
> This radius doesn't persist when the page is scrolled, which is a little weird IMO.

I'm trying to come up with ways to correct this. My first thought is using a pseudo element.